Acd 693 (#44)
Actindo-AG/Documentation: Fulfillment/Overview/01_General.md

# General information

Modern shipping management facilitates the handling of delivery processes and is an important step towards the digitalization of your sales. Whether you work with external fulfillment providers or handle shipping yourself, the Actindo *Fulfillment* module allows you to seamlessly integrate external service providers for fast shipping processing and clear monitoring.   

This documentation refers to *Fulfillment* module version 1.2.2. There may be differences in the UI, features and operation if your *Fulfillment* module is installed in a different version.

## Benefits

The *Fulfillment* module offers the following benefits:

> - integrations to numerous fulfillers - either directly via API or via our Actindo DataHub are readily available

Reviewer: ```suggestion
- integrations to numerous fulfillers &ndash; either directly via API or via the Actindo *DataHub* module are readily available
```

- easy connections to external logistics and fulfillment providers
- automated selection of preferred packaging types and carriers depending on user-defined criteria
- real-time access to updated information on shipment status based on fulfiller's feedback

> [comment]: <> (Check, ob Benefits auf der Webseite aktuell, besonders shadow and consignment warehouses)

Reviewer: ```suggestion
[comment]: <> (Check, ob Benefits auf der Webseite aktuell)
```

## Prerequisites

- The *Order management* module version 4.1.1 or higher is installed.
- The *Actindo DataHub* module version 4.1.0 or higher is installed.
- The *Actindo Work Flow Engine* module version 1.3.0 or higher is installed.
- The *UI Components* plugin version 1.0.1 or higher is installed.
- The *Dashboard* plugin version 1.2.0 or higher is installed.
- The *Countries and languages* plugin version 1.1.0 or higher is installed.

> 

Reviewer: ```suggestion
- The *Currency* plugin version 1.0.9 or higher is installed.
```
